pkg/diagnose/connectivity/endpoint_info.go

```diff
@@ -47,6 +49,15 @@ var (
 	seriesEndpointInfo       = endpointInfo{endpoints.SeriesEndpoint, "POST", emptyPayload}
 	sketchSeriesEndpointInfo = endpointInfo{endpoints.SketchSeriesEndpoint, "POST", emptyPayload}
 
+	// Flare endpoint
+	flareEndpointInfo = endpointInfo{transaction.Endpoint{Route: createFlareEndpoint(), Name: "flare"}, "HEAD", nil}
+
 	endpointsInfo = []endpointInfo{v1SeriesEndpointInfo, v1CheckRunsEndpointInfo, v1MetadataEndpointInfo, v1IntakeEndpointInfo,
-		seriesEndpointInfo, sketchSeriesEndpointInfo, v1ValidateEndpointInfo}
+		seriesEndpointInfo, sketchSeriesEndpointInfo, v1ValidateEndpointInfo, flareEndpointInfo}
 )
+
+func createFlareEndpoint() string {
```

Comment:
Would is be possible to move this to `comp/core/flare/helpers` ?

Reply:
I think so, in send_flare.go seems good to you or should I create a file utils.go ?